starlette_admin/templates/forms/tinymce.html

```diff
@@ -0,0 +1,28 @@
+<div class="{%if error%}{{field.error_class}}{%endif%}">
+    <textarea name="{{field.id}}" id="tinymce-editor" data-bs-toggle="autosize"
+        class="form-control {%if error%}{{field.error_class}}{%endif%}" {{ field.input_params()
+        }}>{%if data%}{{data}}{%endif%}</textarea>
+    {% if field.help_text %}
+    <small class="form-hint">{{field.help_text}}</small>
+    {%endif%}
+</div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tinymce/6.4.0/tinymce.min.js"
+    integrity="sha512-nv2Ftve23IDZqQhji5P2w17Ch88OR37z6tV6djv8U6hcjpRjDXRypN6sXkN6UQo8S+/qf67LVh1a3COduJIG3Q=="
+    crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<script>
+    document.addEventListener("DOMContentLoaded", function () {
+        let options = {
+            selector: '#tinymce-editor',
+            height: 300,
+            menubar: false,
+            statusbar: false,
+            toolbar: 'undo redo | formatselect | ' +
+                'bold italic backcolor | alignleft aligncenter ' +
+                'alignright alignjustify | bullist numlist outdent indent | ' +
+                'removeformat',
+            content_style: 'body { font-family: -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif; font-size: 14px; -webkit-font-smoothing: antialiased; }'
+        }
+        tinyMCE.init(options);
+    })
+</script>
```

Comment:
Put this inside statics/forms.js and use jquery. Something like $(".field-tinymce-editor")

Reply:
The problem with jQuery is that I haven't found a single download that works or a cdn without an api key. There is a TinyCloud cdn without an api key, but then a pop up in TinyMCE in form page appears and that's not right. Do you agree that I use pure JS in `form.js` and add something like this
```javascript
// at the end of form.js
let options = {
  selector: '#tinymce-editor',
  height: 300,
  menubar: false,
  statusbar: false,
  toolbar: 'undo redo | formatselect | ' +
    'bold italic backcolor | alignleft aligncenter ' +
    'alignright alignjustify | bullist numlist outdent indent | ' +
    'removeformat',
  content_style: 'body { font-family: -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif; font-size: 14px; -webkit-font-smoothing: antialiased; }'
}
tinyMCE.init(options);
```